.github/workflows/helmtagbump.yaml

```diff
@@ -71,3 +71,68 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  ui_helm_default_chart_tag:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          ref: develop
+      - name: 'Get Previous tag'
+        id: uiprevioustag
+        uses: "broadinstitute/github-action-get-previous-tag@master"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+      - name: 'Checkout datarepo-helm repo'
+        uses: actions/checkout@v2
+        with:
+          repository: 'broadinstitute/datarepo-helm'
+          token: ${{ secrets.HELM_REPO_TOKEN }}
+          path: datarepo-helm
+      - name: "ui chart default tag find and replace"
+        uses: docker://mikefarah/yq:3
+        with:
+          args: yq w -i datarepo-helm/charts/datarepo-ui/values.yaml image.tag ${{ steps.uiprevioustag.outputs.tag }}"
+      - name: "bump ui chart appVersion"
+        uses: docker://mikefarah/yq:3
+        with:
+          args: yq w -i datarepo-helm/charts/datarepo-ui/Chart.yaml appVersion ${{ steps.uiprevioustag.outputs.tag }}"
```

Comment:
I'm a little confused -  there are two versions in the Chart.yaml, right? "version" and "appVersion". Why do they match right now? they're both 0.0.15. https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo-ui/Chart.yaml#L17

Reply:
```version ==  helm chart version 
and 
appVersion == supposed to be application version 
```
but since we never previously updated the default application version in the values.yaml i never saw a need to change it 


more info here https://helm.sh/docs/topics/charts/